cdispyutils/auth/jwt.py

```diff
@@ -0,0 +1,178 @@
+from collections import OrderedDict
+import json
+
+import flask
+import jwt
+import requests
+
+from .errors import JWTValidationError
+
+
+def refresh_public_keys(user_api=None):
+    """
+    Update the public keys that the Flask app is currently using to validate
+    JWTs.
+
+    Response from ``/keys`` should look like this:
+
+    .. code-block:: javascript
+
+    {
+        "keys": [
+            [
+                "default",
```

Comment:
does this mean when you rotate keys you will change the default key to another one then they share the same kid being `default`?

Reply:
I just put "default" as the example; the key ids will be configured in fence at initialization. "default" isn't actually meaningful, in fact it just defaults to the first key in the list. Keys should not have the same kid.

Reply:
yeah - let's just change the docs to random strings to reduce confusion?

Reply:
yeah, good idea.